lib/addons/prometheus-node-exporter/index.ts

```diff
@@ -0,0 +1,72 @@
+// lib/certmanager_addon.ts
+import { Construct } from 'constructs';
+import merge from "ts-deepmerge";
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace } from "../../utils";
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from "../helm-addon";
+/**
+ * User provided option for the Helm Chart
+ */
+export interface PrometheusNodeExporterAddOnProps extends HelmAddOnUserProps {
+    /**
+     * To Create Namespace using CDK
+     */    
+    createNamespace?: boolean;
+}
+
+/**
+ * Default props to be used when creating the Helm chart
+ */
+const defaultProps: HelmAddOnProps & PrometheusNodeExporterAddOnProps = {
+  name: "prometheus-node-exporter",
+  namespace: "prometheus-node-exporter",
+  chart: "prometheus-node-exporter",
+  version: "2.0.3",

```

Comment:
Why is this version so old? Isn't it already on 4.x versions?

Reply:
Fixed. Please check in the commit.